snap7/util.py

```diff
@@ -522,7 +536,7 @@ def get_bytearray(self) -> bytearray:
             return self._bytearray._bytearray
         return self._bytearray
 
-    def export(self):
+    def export(self) -> Dict:
```

Comment:
This dict type is missing the key and  value types (probably something like `Dict[str, int]`, i cant see that directly.